src/modules/fancyzones/lib/util.cpp

```diff
@@ -66,94 +66,6 @@ namespace FancyZonesUtils
         }
     }
     
-    std::optional<FancyZonesDataTypes::DeviceIdData> ParseDeviceId(const std::wstring& str)
-    {
-        FancyZonesDataTypes::DeviceIdData data;
-
-        std::wstring temp;
-        std::wstringstream wss(str);
-
-        /*
-        Important fix for device info that contains a '_' in the name:
-        1. first search for '#'
-        2. Then split the remaining string by '_'
-        */
-
-        // Step 1: parse the name until the #, then to the '_'
-        if (str.find(L'#') != std::string::npos)
-        {
-            std::getline(wss, temp, L'#');
-
-            data.deviceName = temp;
-
-            if (!std::getline(wss, temp, L'_'))
-            {
-                return std::nullopt;
-            }
-
-            data.deviceName += L"#" + temp;
-        }
-        else if(std::getline(wss, temp, L'_') && !temp.empty())
-        {
-            data.deviceName = temp;
-        }
-        else
-        {
-            return std::nullopt;
-        }
-
-        // Step 2: parse the rest of the id
-        std::vector<std::wstring> parts;
-        while (std::getline(wss, temp, L'_'))
-        {
-            parts.push_back(temp);
-        }
-
-        if (parts.size() != 3 && parts.size() != 4)
-        {
-            return std::nullopt;
-        }
-
-        /*
-        Refer to ZoneWindowUtils::GenerateUniqueId parts contain:
```

Comment:
This function is called `FancyZonesUtils::GenerateUniqueId`, so
let's update the comment.